tests/system_tests/conftest.py

```diff
@@ -98,6 +99,42 @@ def user(mocker, backend_fixture_factory) -> Iterator[str]:
     yield username
 
 
+@dataclass
+class UserOrg:
+    username: str
+    organization_name: str
+
+
+@pytest.fixture
+def user_in_org(mocker, backend_fixture_factory) -> Iterator[UserOrg]:
```

Comment:
this is just a special case of the user in orgs, maybe just implement something general that creates any number of orgs instead of having these 2 almost identical fixtures